packages/eas-cli/schema/metadata-0.json

```diff
@@ -606,11 +606,10 @@
       "description": "Configuration that is specific to the Apple App Store.",
       "additionalProperties": false,
       "properties": {
-        "advisory": {
-          "$ref": "#/definitions/apple/AppleAdvisory"
-        },
-        "categories": {
-          "$ref": "#/definitions/apple/AppleCategory"
+        "version": {
+          "type": "string",
+          "description": "The version string to edit, or create, if it doesn't exist.",
```

Comment:
Maybe an example of the version string could be helpful so users don't confuse it with the build number.

Reply:
Done, default behavior is using the latest editable version